Entities/AHuman.cpp

```diff
@@ -223,56 +220,56 @@ int AHuman::Create(const AHuman &reference)
 //                  is called. If the property isn't recognized by any of the base classes,
 //                  false is returned, and the reader's position is untouched.
 
-int AHuman::ReadProperty(std::string propName, Reader &reader)
-{
-	if (propName == "ThrowPrepTime")
-		reader >> m_ThrowPrepTime;
-	else if (propName == "Head")
-    {
-        delete m_pHead;
+int AHuman::ReadProperty(std::string propName, Reader &reader) {
+    if (propName == "ThrowPrepTime") {
+        reader >> m_ThrowPrepTime;
+    } else if (propName == "Head") {
+        RemoveAttachable(m_pHead);
         m_pHead = new Attachable;
         reader >> m_pHead;
-		if (!m_pHead->IsDamageMultiplierRedefined())
-			m_pHead->SetDamageMultiplier(5);
-    }
-    else if (propName == "Jetpack")
-    {
-        delete m_pJetpack;
+        AddAttachable(m_pHead);
+        if (!m_pHead->GetDamageMultiplierSetInINI()) { m_pHead->SetDamageMultiplier(5.0F); }
+        m_pHead->SetInheritsRotAngle(false);
+    } else if (propName == "Jetpack") {
+        RemoveAttachable(m_pJetpack);
         m_pJetpack = new AEmitter;
         reader >> m_pJetpack;
-    }
-    else if (propName == "JumpTime")
-    {
+        AddAttachable(m_pJetpack);
+        if (!m_pJetpack->GetDamageMultiplierSetInINI()) { m_pJetpack->SetDamageMultiplier(0.0F); }
+        m_pJetpack->SetApplyTransferredForcesAtOffset(false);
+        m_pJetpack->SetDeleteWhenRemovedFromParent(true);
```

Comment:
I left it this way to maintain current behaviour. I actually tried turning it off to see how it goes and it's not great, jetpacks are generally not set up to exist on their own, so they don't break or anything, and just kinda sit there.
I'd say this is another case where the answer is to stick on a script that unsets this during its Create.

Reply:
What do you mean `jetpacks are generally not set up to exist on their own`?
The jetpack preset that's being used everywhere is just set to `GetsHitByMOs = 0` and doesn't have any wound/gib definitions.
There's nothing stopping a jetpack from being destructible.

Reply:
Sorry, what I really meant is that it looked wonky to me, especially with the browncoat jetpack cause it's big.
Turn it off and get some guys gibbed, see what you think. If it seems better without this then I've got no problem turning it off.

Reply:
You know perfectly well that I cba doing that.
The property is exposed to ini so it can be set true to whatever is relevant so you can remove the line.
I'll add the property to all the jetpacks and we (4zk) can figure out what jetpacks we want detachable/destructible later.

Keep this unresolved so I don't forget.

Reply:
lol you lazy bastard.
Sure, I'll turn it off and leave that ini followup to you guys. Might screw with mods a little but not a hugely noticeable change.